tools/test_deleted_renamed_referenced_files

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh -e
+
+FILES="${@}"
+success=1
+for FILE in $FILES; do
+    if [ -f "$FILE" ]; then
+        # if file exists it means it was renamed, and then original file name is retrieved by git log
+        FILE=$(git log --follow -p $FILE | grep 'rename from test' | awk '{print $3}')
+    fi
+    if [ -n "$(echo $FILE | grep '.pm$')" ]; then
+    # In case file is a module, module name appears in scheduling files excluding 'tests/' and file extension
+    filename=$(echo $FILE | cut -f 2- -d '/' | cut -f 1 -d '.')
```

Comment:
I guess we could use basename/dirname utils here

Reply:
hmm... but we don't use basename or dirname in schedule. We just cut the tests/ and the .pm 
tests/installation/partitioning.pm -> installation/partitioning

Reply:
Ah, it's variable name which confused me, I thought that you are extracting just file name.
Also, consider this example: 
```sh
echo "inst/pam.pam.pm" | cut -f 1 -d '.'
Output: inst/pam
```
As we know that we won't have extensions like `tar.gz` we could simply remove `.pm` from the end.

With sed:
```
echo tests/installation/part.pm.pm | sed -E "s/^tests\/(.*)\.pm$/\1/g"
```

So in this case I would suggest at least renaming `filename` variable to `module`.
Could you also please fix indentation in the if clause?
Also, keep in mind that normally you put `then` on next line and semicolon syntax is more for one liners ( see examples here: https://www.tutorialspoint.com/unix/if-else-statement.htm )

Reply:
> So in this case I would suggest at least renaming `filename` variable to `module`.

Initially it was 'module' in Joaquin's script, I changed it to 'filename' because now the variable might represent either a module or a test_data file. After your comment I changed it to file_to _verify in order to avoid confusion. WDYT?

Rest, done.

Reply:
Good point. I guess we can leave current name then. Cheers!